File: datamodel/src/main/java/datamodel/clinical/ToxicitySource.java
package datamodel.clinical;

import org.jetbrains.annotations.NotNull;

public enum ToxicitySource {
    QUESTIONNAIRE("Questionnaire"),
    EHR("EHR");

    @NotNull
    private final String display;

    ToxicitySource(@NotNull final String display) {
        this.display = display;
    }

    @NotNull
    public String display() {
        return display;
    }
}

File: algo/src/main/kotlin/com/hartwig/actin/datamodel/clinical/ToxicitySource.kt
package com.hartwig.actin.datamodel.clinical

import com.hartwig.actin.datamodel.Displayable

enum class ToxicitySource(private val display: String) : Displayable {
    QUESTIONNAIRE("Questionnaire"),
    EHR("EHR");

    override fun display(): String {
        return display
    }
}
